catalog/neon-prod-earthengine/projects_neon-prod-earthengine_assets_HSI_REFL_001.jsonnet

```diff
@@ -24,31 +24,33 @@ local units = import 'units.libsonnet';
   title: 'NEON Directional Surface Reflectance',
   'gee:type': ee_const.gee_type.image_collection,
   description: |||
-    The NEON AOP surface directional reflectance data is an orthorectified 
-    hyperspectral VSWIR (visible to shortwave infrared) data product, containing 
+    The NEON AOP surface bidirectional reflectance data is a hyperspectral 
```

Comment:
But I thought this is the directional reflectance dataset and HSI_REFL_002 is the bidirectional reflectance one, no?

Reply:
You're correct! Just fixed in [0aab173](https://github.com/google/earthengine-catalog/pull/1111/commits/0aab1738a3c63dd20ba01becce3370e4271e564e)